va/validation-authority.go

```diff
@@ -260,7 +261,9 @@ func (va *ValidationAuthorityImpl) fetchHTTP(ctx context.Context, identifier cor
 		}
 	}
 
-	body, err := ioutil.ReadAll(httpResponse.Body)
+	// payload should be ~87 bytes, give a little leeway
+	// (32 byte b64 encoded token + . + 32 byte b64 encoded key fingerprint)
+	body, err := ioutil.ReadAll(&io.LimitedReader{R: httpResponse.Body, N: 174})
```

Comment:
174 is oddly specific, even if we const-ize it. I'd rather this either be a power of 2, or represented as `87 + __`.